### PR TITLE
JSON - Study - Response

### PR DESCRIPTION
**Pull request by BenGitsCode**
I had some trouble understanding this one but I'm pretty sure it's a lot easier
and straight-forward than it sounds.

### CODE REVIEW

File: study.md [hunk 1]
```diff
@@ -38,29 +38,83 @@ As it makes sense to you please write the following in JSON format:
...
+    "gender": "female",
+  }
+  "roommates":
+  {
+    "name":"Dave",
+    "name": "Miller"
+  }
+  "loves":
+  {
+    ["The Red Sox"], [" Patriots"], [" Bruins"]
```
Comment (RealWeeks) [inline study.md:57]: why separate arrays for each one?

File: study.md [hunk 2]
```diff
@@ -38,29 +38,83 @@ As it makes sense to you please write the following in JSON format:
...
+    {
+      "type": "cat",
+      "name: "Mr Kitty",
+      "gender": "female",
+    }
+    "roommates": {
+      "name":"Dave" {
+        "pets": {
+          "goldfish": {
+            "name":"Bob", "color": "red",
```
Comment (RealWeeks) [inline study.md:87]: you may want to break each fish up into a separate "object"

**Comment by RealWeeks** [comment]
Nice job, good reading.  It is much for straightforward once you begin to work with it.

File: study.md (hunk 1 above)
Comment (BenGitsCode) [inline study.md:57]: I was under the impression that each "object" that breaks down into new types of properties had to be broken up within the object that contained it all. Should it be more like this?: {"Jason's Things"
 +
-  "pet": "cat"
-  {
-    "type": "cat",
-    "name: "Mr Kitty",
-    "gender": "female",
-  }
-  "roommates":
-  {
-    "name":"Dave",
-    "name": "Miller"
-  }
-  "loves":
-  {
-    "The Red Sox", " Patriots", " Bruins"

I'm correct in breaking things like loves, pets and roommates into separate parts right?

Comment (RealWeeks) [inline study.md:57]: correct.  I would do something like "loves": ["sox", "pats" ...]

**Pull request closed without merging**